src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -1,41 +1,100 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './WebcamCapture.css';
 import Webcam from 'react-webcam';
 import Notification from './../Notification/Notification';
 import PropTypes from 'prop-types';
+import Model from './../../utils/model';
 
-const WebcamCapture = props => {
-	const height = 400;
-	const width = 400;
+const ML_THRESHOLD = 0.06;
+const height = 400;
+const width = 400;
 
-	const videoConstraints = {
-		width,
-		height,
-		facingMode: 'user'
+const videoConstraints = {
+	width,
+	height,
+	facingMode: 'user'
+};
+
+class WebcamCapture extends Component {
+	state = {
+		isDetecting: true,
+		cameraConnected: false
 	};
 
-	if (props.cameraConnected) {
+	constructor(props) {
+		super(props);
+		this.webcam = React.createRef();
```

Comment:
We could define the ref as a class property (like we do with state) and move the model loading to `componentDidMount` to keep our side effects in line with the module lifecycle

e.g.

```
class WebcamCaputre extends Component {
  webcam = React.createRef()

  componentDidMount() {
    this.model = new Model();
    this.model.load();
  }
}
```